docs/tutorials/02_converters_for_quadratic_programs.ipynb

```diff
@@ -266,51 +242,32 @@
   {
    "cell_type": "code",
    "execution_count": 7,
-   "metadata": {
-    "tags": []
-   },
+   "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "\\ This file has been generated by DOcplex\n",
-      "\\ ENCODING=ISO-8859-1\n",
-      "\\Problem name: CPLEX\n",
+      "Problem name: \n",
       "\n",
       "Maximize\n",
-      " obj: 2 x + y + z@0 + 2 z@1 + 4 z@2\n",
-      "Subject To\n",
-      " xyz_leq: x + y + z@0 + 2 z@1 + 4 z@2 + xyz_leq@int_slack@0\n",
-      "          + 2 xyz_leq@int_slack@1 + 2 xyz_leq@int_slack@2 = 5\n",
-      " xyz_geq: x + y + z@0 + 2 z@1 + 4 z@2 - xyz_geq@int_slack@0\n",
-      "          - 2 xyz_geq@int_slack@1 - 3 xyz_geq@int_slack@2 = 3\n",
+      "  2*x + y + z@0 + 2*z@1 + 4*z@2\n",
       "\n",
-      "Bounds\n",
-      " 0 <= x <= 1\n",
-      " 0 <= y <= 1\n",
-      " 0 <= z@0 <= 1\n",
-      " 0 <= z@1 <= 1\n",
-      " 0 <= z@2 <= 1\n",
-      " 0 <= xyz_leq@int_slack@0 <= 1\n",
-      " 0 <= xyz_leq@int_slack@1 <= 1\n",
-      " 0 <= xyz_leq@int_slack@2 <= 1\n",
-      " 0 <= xyz_geq@int_slack@0 <= 1\n",
-      " 0 <= xyz_geq@int_slack@1 <= 1\n",
-      " 0 <= xyz_geq@int_slack@2 <= 1\n",
+      "Subject to\n",
+      "  Linear constraints (2)\n",
+      "    x + xyz_leq@int_slack@0 + 2*xyz_leq@int_slack@1 + 2*xyz_leq@int_slack@2 + y + z@0 + 2*z@1 + 4*z@2 == 5  'xyz_leq'\n",
+      "    x - xyz_geq@int_slack@0 - 2*xyz_geq@int_slack@1 - 3*xyz_geq@int_slack@2 + y + z@0 + 2*z@1 + 4*z@2 == 3  'xyz_geq'\n",
```

Comment:
These end up wrapping in the html docs so the end result is not as nice. Not sure there is a great deal one can do about this. I imagine that we could have the prettyprint wrap things nicely at a given limit, so it determines/manages the word wrapping, but maybe that's too much work for now.
![image](https://user-images.githubusercontent.com/40241007/168171473-8497d726-552e-470c-8185-507522099f31.png)

Reply:
Thanks. Wrapping lines might be a good option to improve the appearance. I will try it.

Reply:
I'm trying `wrap` option. Here are some results with different wrap values. What do you think of these appearance? If OK, I will write docstrings  and unit tests.

wrap = 100
```
Problem name: problem 1

Minimize
   43*lin_GE@int_slack@0^2 + 86*lin_GE@int_slack@0*lin_GE@int_slack@1 + 43*lin_GE@int_slack@1^2
   - 86*x@0*lin_GE@int_slack@0 - 86*x@0*lin_GE@int_slack@1 + 215*x@0^2 + 860*x@0*x@1 - 344*x@0*y@0
   - 688*x@0*y@1 - x@0*z@0 - 2*x@0*z@1 - 3*x@0*z@2 - 172*x@1*lin_GE@int_slack@0 - 172*x@1*lin_GE@int_slack@1
   + 860*x@1^2 - 688*x@1*y@0 - 1376*x@1*y@1 - 2*x@1*z@0 - 4*x@1*z@1 - 6*x@1*z@2 + 172*y@0*lin_GE@int_slack@0
   + 172*y@0*lin_GE@int_slack@1 + 215*y@0^2 + 860*y@0*y@1 + 344*y@1*lin_GE@int_slack@0
   + 344*y@1*lin_GE@int_slack@1 + 860*y@1^2 + 86*lin_GE@int_slack@0 + 86*lin_GE@int_slack@1 - 427*x@0
   - 854*x@1 + 347*y@0 + 694*y@1 + z@0 + 2*z@1 + 3*z@2 + 212 

Subject to
  No constraints

  Binary variables (12)
     x@0 x@1 y@0 y@1 z@0 z@1 z@2 xxx3 xxx4 xxx5 lin_GE@int_slack@0 lin_GE@int_slack@1
```

wrap = 50
```
Problem name: problem 1

Minimize
   43*lin_GE@int_slack@0^2
   + 86*lin_GE@int_slack@0*lin_GE@int_slack@1
   + 43*lin_GE@int_slack@1^2
   - 86*x@0*lin_GE@int_slack@0
   - 86*x@0*lin_GE@int_slack@1 + 215*x@0^2
   + 860*x@0*x@1 - 344*x@0*y@0 - 688*x@0*y@1 - x@0*z@0
   - 2*x@0*z@1 - 3*x@0*z@2 - 172*x@1*lin_GE@int_slack@0
   - 172*x@1*lin_GE@int_slack@1 + 860*x@1^2
   - 688*x@1*y@0 - 1376*x@1*y@1 - 2*x@1*z@0 - 4*x@1*z@1
   - 6*x@1*z@2 + 172*y@0*lin_GE@int_slack@0
   + 172*y@0*lin_GE@int_slack@1 + 215*y@0^2
   + 860*y@0*y@1 + 344*y@1*lin_GE@int_slack@0
   + 344*y@1*lin_GE@int_slack@1 + 860*y@1^2
   + 86*lin_GE@int_slack@0 + 86*lin_GE@int_slack@1
   - 427*x@0 - 854*x@1 + 347*y@0 + 694*y@1 + z@0 + 2*z@1
   + 3*z@2 + 212 

Subject to
  No constraints

  Binary variables (12)
     x@0 x@1 y@0 y@1 z@0 z@1 z@2 xxx3 xxx4 xxx5
     lin_GE@int_slack@0 lin_GE@int_slack@1
```

Reply:
Looks good to me. Given you show different values above I assume you will wrap at some default value and let the user change that if they want for some reason.

Reply:
Thanks, I will tune the default value looking at the jupyter notebook output and let users change that if they wish.